### PR TITLE
Update masks.yml

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Fix merc gas mask to match all other masks stats

## Why / Balance
Let me drip

## How to test
N/A

## Media
N/A

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Masks/masks.yml [hunk 1]
```diff
@@ -397,8 +397,8 @@
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.90
-        Slash: 0.90
+        Blunt: 0.95 # Frontier 0.90<0.95
+        Slash: 0.95 # Frontier 0.90<0.95
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Clothing/Masks/masks.yml:401]: Should the swat gas mask also be brought into line with this?

Comment (dvir001) [reply Resources/Prototypes/Entities/Clothing/Masks/masks.yml:401]: sure

**Comment by whatston3** [comment]
Assuming the goal is to promote choice of mask by leveling the field, this makes sense to me.

**Review by whatston3 — APPROVED** [review]

**Comment by whatston3** [comment]
muh changelog

**Pull request merged**